src/main/java/com/microsoft/aad/adal4j/AdalIntegratedAuthorizationGrant.java

```diff
@@ -0,0 +1,28 @@
+package com.microsoft.aad.adal4j;
+
+import java.util.Map;
+
+public class AdalIntegratedAuthorizationGrant implements AdalAuthorizationGrant {
```

Comment:
Integrated auth is not one of the OAuth grant types, so why is it implementing the AdalAuthorizationGrant interface?   Again, this may be my lack of understanding of what AdalAuthorizationGrant represents

Reply:
AdalAuthorizationGrant represent all Adal grants